README.md

```diff
@@ -118,3 +118,12 @@ has the subkeys you want to test against (`name`, `housenumber`…).
 Optional keys: `limit`, `lang`, `lat` and `lon`, `skip`.
 You can add categories to your test by using the key `mark` (which expects a
 list), that you can then run with `-m yourmarker`.
+
+## License
+
+Geocoder-tester is available under a MIT license. See LICENSE.txt for more
+information.
+
+As a special exception, the test cases under `geocoder_tester/world/` are
+considered to be in the public domain. You may use them without any
+restrictions.
```

Comment:
I've used 3rd parties datasets (Wikidata, OpenFlights, etc) to create some test cases, so some of them may not be considered to be in the public domain. For instance: https://github.com/geocoders/geocoder-tester/blob/master/geocoder_tester/world/poi/license.md
Maybe add "unless otherwise specified in the test directory readme" here ?